src/routes/SinglePost/components/SinglePostPage.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Button,
+  Container,
+  Typography,
+  Paper,
+  makeStyles,
+} from '@material-ui/core';
+import { Redirect, useRouteMatch, Link } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { BLOG_PATH } from 'constants/paths';
+import { ChevronLeft as BackIcon } from '@material-ui/icons';
+import styles from './SinglePostPage.styles';
+
+const useStyles = makeStyles(styles);
+
+function SinglePostPage() {
+  const classes = useStyles();
+  const [post, setPost] = useState({});
+  const [loaded, setLoaded] = useState(false);
```

Comment:
What is the reason for both the loaded and loading states? Couldn't one state handle both? Guessing this is from the blog changes though, so it can be left, just asking

Reply:
It can.  I think this whole piece can be replaced with the reducer like you had previously suggested to Mauricio.  So, I was thinking that these pages will be redone later anyways and  didn't worry too much about these for now.